cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/type/logviewer/logparser/LogParserStrategy.kt

```diff
@@ -0,0 +1,8 @@
+package com.cognifide.cogboard.widget.type.logviewer.logparser
+
+import main.kotlin.com.cognifide.cogboard.logStorage.Log
+
+interface LogParserStrategy {
+    val variableFields: List<String>
+    fun parseLine(line: String): Log?
```

Comment:
Question/fix: If the `line` cannot be null why `Log` can?

Reply:
Changed, the parser is now expected to always return a log.